tests/Merchant_tests.py

```diff
@@ -1,35 +1,85 @@
 import unittest
-from os.path import dirname, join
+from unittest import mock
+from os.path import dirname, join, abspath
+from os import path
+import json
 from ddt import ddt, unpack, data
 from fattmerchant.Merchant import Merchant
 from fattmerchant.Customer import Customer
+from fattmerchant.utils import compare_json_data
 import logging
 
+
 logger = logging.getLogger(__name__)
+test_path = dirname(abspath(__file__))
+static_data = join(test_path, "static_data")
+
+
+def mocked_requests_put(*args, **kwargs):
```

Comment:
I believe most of my comments from the closed PR still stand for this PR, but since we need to get this in ASAP, I can understand if we just let this in anyways for now and clean it up a little bit later.